stipple.py

```diff
@@ -0,0 +1,33 @@
+# Import the dependencies
+import argparse
+import PIL
+import random
+from PIL import Image
+from PIL import ImageColor
+import math
+import cv2
```

Comment:
Why is this required?

Reply:
It is not, I was looking for some direct functionality so I had imported it but forget to remove.But now its not there